Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Dockerfile (For Development)
+FROM node:18-alpine
+WORKDIR /app
+COPY package*.json ./
+RUN npm ci
```

Comment:
```suggestion
RUN npm install
```

Reply:
@LeoQiu981112 also please add this to the `next.config.js` in order to make hot reload work
```
webpackDevMiddleware: (config) => {
    config.watchOptions = {
      poll: 1000,
      aggregateTimeout: 300,
    };
    return config;
  },
  ```

Reply:
@alchuu00 , do you know why that is necessary?

Reply:
@SamHuo213 without this part of code the hot reloading wasn't working. 

So this part of code is telling webpack to check for changes in the codebase every 1 second (poll: 1000). If any changes are detected during this polling interval, webpack will initiate a rebuild of the application.